crates/subspace-farmer/Cargo.toml

```diff
@@ -56,7 +56,7 @@ zeroize = "1.6.0"
 
 # The only triple tested and confirmed as working in `jemallocator` crate is `x86_64-unknown-linux-gnu`
 [target.'cfg(all(target_arch = "x86_64", target_vendor = "unknown", target_os = "linux", target_env = "gnu"))'.dependencies]
-jemallocator = "0.5.0"
+tikv-jemallocator  = "0.5.0"
```

Comment:
Turns out this crate is released as `tikv-jemallocator` and just `jemallocator`. Unfortunately, one of the frontier's deps', rocksdb, snappy, depends on `tikv-*` version of libs. This lead to same native lib link to both version.
more here - https://github.com/tikv/jemallocator/releases/tag/0.5.0

Reply:
We were trying to hard to get rid of rocksdb, and now we depend on it again? Is it a required dependency rather than optional? Can we make it optional like I made it optional in Substrate?

Reply:
I dont like it as well but frontier supports both the DBs.
We though I made it optional, cargo still pull the optional deps AFAIK. Correct me if I'm wrong here. 
So when that dep pulls `librocksdb-sys`, which internally pull `tikv-jemalloc` which was already linked though `subspace-farmer`.

Let me try if we can avoid that again

Reply:
> We though I made it optional, cargo still pull the optional deps AFAIK. Correct me if I'm wrong here.

It is optional in Substrate and we don't pull it because we don't use it, not even in our lock file.